paasta_tools/clusterman.py

```diff
@@ -0,0 +1,17 @@
+import staticconf
```

Comment:
I'm still surprised this works, as we are not including it in any requirements.txt. It must be some transitive dependency.